Dockerfile

```diff
@@ -125,13 +125,13 @@ RUN --mount=type=cache,target=/root/.cache/go-build \
     GOARCH=${TARGETARCH:-amd64} \
     CGO_ENABLED=0 \
     GO111MODULE=on \
-    go build -a -o /go/bin/server ./proxy/
+    go mod tidy && go build -a -o /go/bin/server ./proxy/
```

Comment:
why do a `go mod tidy` here?  I think we should just keep the `go.mod` and `go.sum` files "tidy" in the main repo